CodePush.m

```diff
@@ -523,55 +493,42 @@ - (void)savePendingUpdate:(NSString *)packageHash
                                 rejecter:(RCTPromiseRejectBlock)reject)
 {
     [CodePush removePendingUpdate];
-    resolve([NSNull null]);
+    resolve(nil);
 }
 
 /*
- * This method is checks if a new status update exists (new version was installed, 
+ * This method is checks if a new status update exists (new version was installed,
  * or an update failed) and return its details (version label, status).
  */
 RCT_EXPORT_METHOD(getNewStatusReport:(RCTPromiseResolveBlock)resolve
                             rejecter:(RCTPromiseRejectBlock)reject)
 {
     if (needToReportRollback) {
```

Comment:
Should we wrap this entire method in a call to "dispatch_async" (just like "installUpdate") so that we don't synchronously tie up the native module thread?